docs/development/local_setup.md

```diff
@@ -376,19 +382,19 @@ to operate against your local running Gardener API Server.
 
 > Note: It may take several seconds until the Gardener API server has been started and is available. `No resources found` is the expected result of our initial development setup.
 
-### Create a Shoot
+# Create a Shoot
 
 The steps below describe the general process of creating a Shoot. Have in mind that the steps do not provide full example manifests. The reader needs to check the provider documentation and adapt the manifests accordingly.
 
-#### 1. Copy the example manifests to dev/
```

Comment:
I think you're absolutely right about the consistency bit: when using numbered headings, use them everywhere. 

However, I'm not sure about the 'when to use numbered headings' part. I don't think people use numbered headings to indicate an order of steps (first do this, then that), like you would do in an enumeration – I don't think people looking at the document are confused in which order to follow these steps, but work through it from top to bottom. It's rather to help the reader understand a hierarchy in your document: Heading 1 has sub-heading 1.1 to 1.5, then follows heading 2, etc. – this helps to open and close context in your head while reading stuff. Typically, headings are also too far apart to serve the purpose of enumerating steps, I think.

I didn't really find a good source on 'when to use numbered headings' with clear recommendations, though, it might also depend on the kind of document. I choose an example from the official SAP docs which shows the distinction I was trying to make above: https://help.sap.com/viewer/4287333baaa6413a8ece0a8ed1196af4/LATEST/en-US/25f37ca392064adaabb317c7513f1dc1.html
* The headings are not numbered, but summarize what's happenening there
* You might have an enumeration of steps inside a section (first do this, then that)

Lastly, there's a technical inconvenience with numbered headings: you have to maintain them manually.

Reply:
I don't insist to have headings. We can for example have numbered steps that are not headings. I think that having numbered steps is kind of a fundamental thing for such type of guides.

> I don't think people looking at the document are confused in which order to follow these steps

I don't agree with this.

Reply:
I brought back numbered headings. I tried to use github's logic for automatic ordered lists with 
```
1. first this
1. then this
```
But it didn't work, presumably because the steps contain one or more paragraphs each.